src/sentry/models/sentryapp.py

```diff
@@ -119,6 +119,9 @@ class SentryApp(ParanoidModel, HasApiScopes):
     date_updated = models.DateTimeField(default=timezone.now)
     date_published = models.DateTimeField(null=True, blank=True)
 
+    creator_user_id = models.IntegerField(null=True)
```

Comment:
would we want an index on this?

Reply:
@MeredithAnya I doubt we'd want to query all sentry apps created by a user (we'd just want the join).